tests/ffi/qm_nested/test.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../common/prepare.sh
+
+# Declare global variables and initialize to 0
+NESTED_NAME=0
```

Comment:
As we are expecting string, how about init it as "" ? The same for NESTED_STATUS.